UserTools/Configuration/Configuration.cpp

```diff
@@ -72,15 +74,31 @@ void Configuration::Thread(Thread_args* arg){
 
 bool Configuration::LoadConfig(){
 
+  //m_data->services->SendLog("Loading configuration "+std::to_string(m_data->run_configuration)+" from DB" , 1);
+  Log("Loading configuration "+std::to_string(m_data->run_configuration)+" from DB" , 1);
   std::string config_json="";
+  std::string response="";
   if(m_data->services->GetRunDeviceConfig(config_json, m_data->run_configuration)){
+    
     m_data->vars.JsonParser(config_json);
-    m_data->change_config=true;
-    InitialiseConfiguration("");
+    
+    InitialiseConfiguration(m_configfile);
     ExportConfiguration();
-    return true;
+    std::cout<<m_tool_name<<" m_variables:\n--------------\n";
+    m_variables.Print();
+    std::cout<<"--------------\n";
+    
+    m_data->change_config=true;
+    
+  } else {
+    
+    m_data->services->SendLog("ERROR "+m_tool_name+": Failed to load config from DB with error '"+config_json+"'" , 0);
+    std::clog<<"sending alarm of error getting DB config"<<std::endl;
+    bool ok = m_data->services->SendAlarm("ERROR "+m_tool_name+": Failed to load config from DB with error '"+config_json+"'");
+    std::clog<<"sendAlarm of error getting DB config returned "<<ok<<std::endl;
+    return false;
+    
   }
-  m_data->services->SendLog("ERROR DAQ Configuration: Failed to load config from DB" , 0);
-  m_data->services->SendAlarm("ERROR DAQ Configuration: Failed to load config from DB");
-  return false;
+  
```

Comment:
not sure why you removed these

Reply:
they were replaced by ones with more information on the failure, as part of the else clause.